num-generator/src/main.ts

```diff
@@ -0,0 +1,28 @@
+import { NestFactory } from '@nestjs/core';
```

Comment:
the same sort as above, I will not mention this anymore just follow that in all places